.github/actions/prepare-frontend/action.yml

```diff
@@ -2,20 +2,10 @@ name: Prepare front-end environment
 runs:
   using: "composite"
   steps:
-    - name: Prepare Node.js
-      uses: actions/setup-node@v2
-      with:
-        node-version: 14.x
-        cache: 'yarn'
+    - name: Prepare node environment
+      uses: ./.github/actions/prepare-node
     - name: Get M2 cache
       uses: actions/cache@v2
       with:
         path: ~/.m2
         key: ${{ runner.os }}-cljs-${{ hashFiles('**/shadow-cljs.edn') }}
-    - name: Get node_modules cache
```

Comment:
Do we intentionally want to lose this `node_modules` caching?

Reply:
It's now on the prepare-node, that is used on the frontend

Reply:
I don't understand. Caching `~/.cache/yarn`, which `actions/setup-node` does, is different than caching `node_modules`. See #19202 for details.

Reply:
Sorry I was misunderstanding things. This may not make sense